spec/segment/analytics/e2e_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+module Segment
+  # End-to-end tests that send events to a segment source and verifies that a
+  # webhook connected to the source (configured manually via the app) is able
+  # to receive the data sent by this library.
+  describe 'End-to-end tests', e2e: true do
+    WRITE_KEY = 'qhdMksLsQTi9MES3CHyzsWRRt4ub5VM6'
+    RUNSCOPE_BUCKET_KEY = 'pwb8mcmfks0f'
```

Comment:
Can you add a comment mentioning this bucket can be viewed at `https://www.runscope.com/stream/umkvkgv7ndby`

Reply:
(`umkvkgv7ndby` and not `pwb8mcmfks0f` - sorry about the mixup!)